### DISCUSSION TITLE
Need some help installing this in the RE games

### DISCUSSION THREAD

**Discussion by ghostspider89** [praydog/REFramework#756] [General Questions]
I love this mod when I can get it working but figuring out which files to download have always been an absolute nightmare. I download the three files needed:

the latest dinput8 file as provided from the github page
the latest pdperfplugin off nexus mods
and the dlss version 2.5.1

every single time i drag these into the game folder and it never works. the menu pops up but theres no upscaler option. it always turns out i need a very specific version of one of the files to install. 

none of it has ever made any sense. you would think installing the latest version of the mod would work but it never does.

so could someobody please help me and explain what I need to do exactly to get this mod to work? what do i need to look out for when downloading files so i know im downloading the right version? as it stands none of it makes sense.

perhaps some instructions with the correct links for the proper versions would help. as it stands its impossible to get running. Id appreciate any help

**Comment by praydog** [comment]
You need the latest `pd-upscaler` branch from this page: https://github.com/praydog/REFramework/actions

Do not download the `master` branch or anything else. This feature is not fully released so instructions aren't provided in a proper way at this time.

**Reply by RaggyD** [reply]
Hi sometimes I can be dense so I'm just...I feel like surely I'm doing something wrong but I can't see what.  I've got the same issue, no option for temporal upscaling, but only in RE8.  It works fine for me in RE4R.  I used dinput8.dll from the 1.479 nightly file.  If there's something more specific under the pd-upscaler branch I'm not sure what it is.  

I appreciate you taking the time to reply and try to help. :)

Edit: derp it was right at the top of the list.  Autism sometimes I swear.  Sometimes I'm like lightning other times I'm like mud.  :P  It worked yay!
